akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -24,12 +24,12 @@ Element
   downstream. Buffer sizes are always expressed as number of elements independently form the actual size of the elements.
 Back-pressure
   A means of flow-control, a way for consumers of data to notify a producer about their current availability, effectively
-  slowing down the upstream source to match their consumption speeds.
-  In the context of Akka Streams back-pressure is always understood as *non-blocking* and *asynchronous*
+  slowing down the upstream producer to match their consumption speeds.
```

Comment:
I think in this paragraph it's not possible to avoid the terms "consume(r)" and producer. Source and friends are only introduced later. So let's keep it.